qa/TL1_python-self-test-failure/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+# used pip packages
+pip_packages='${python_test_runner_package}'
+target_dir=./dali/test/python
+
+test_body() {
+    ${python_new_invoke_test} --config unittest_failure.cfg test_trigger_failure.py
```

Comment:
How about just `python -c "raise TypeError()"`?

Reply:
I can add another one, but I think it's better to try to test something that is as close as possible to the proper testing behaviour.

Reply:
OK

Reply:
Mixed feelings here. I'd perhaps go as far the other way as possible and made the `test_body` execute something trivially failing, such as `rm /tmp/this_file_does_not_exist`.